File: Exercicio05_p3/Cliente.java

import java.util.ArrayList;
import java.util.Date;
import java.util.List;

public class Cliente {

    int id;
    String nome;
    String cpf;
    Date dataNascimento;
    List<Conta> contas;

    public Cliente(int id, String nome, String cpf, Date dataNascimento) {
        this.id = id;
        this.nome = nome;
        this.cpf = cpf;
        this.dataNascimento = dataNascimento;
        this.contas = new ArrayList<>();
    }
}

File: Exercicio_05/Cliente.ts
import Conta from "./Conta";

class Cliente {

    id: number;
    nome: string;
    cpf: string;
    dataNascimento: Date;
    contas: Conta[];

    constructor(id: number, nome: string, cpf: string, dataNascimento: Date) {
        this.id = id;
        this.nome = nome;
        this.cpf = cpf;
        this.dataNascimento = dataNascimento;
        this.contas = [];
    }
}

export default Cliente;